product-service-description/compliance-as-a-service/features/SQLite-Storage.md

```diff
@@ -1,6 +1,6 @@
 # SQLite Storage
 
-Die Verwendung des SQLite Storage-Providers ermöglicht den Betrieb der Middleware mit einer lokalen, dateibasierten Datenbank. Die Middleware speichert dabei sämtliche beim Betrieb der Queue verarbeiteten Daten sowieso alle Konfigurationsdaten direkt in der Datenbank ab. 
+Die Verwendung des SQLite Storage-Providers ermöglicht den Betrieb der Middleware mit einer lokalen, dateibasierten Datenbank. Die Middleware speichert dabei sämtliche beim Betrieb der Queue verarbeiteten Daten sowie alle Konfigurationsdaten direkt in der Datenbank ab. 
```

Comment:
Hat sich dass zu den anderen Märkten geändert? Dort gibt es doch config-files die über den Launcher geladen/erzeugt werden und nicht in die Datenbank kommen.

Reply:
Nein, das schon immer so. Die Config wird im Helipad erzeugt, der Launcher "injected" das dann in die Middleware, und die persistiert diese Daten in der Datenbank. Das ist nötig, weil man die Config ja immer updaten kann, sich gewisse Daten aber nicht mehr ändern dürfen sobald die Queue in Betrieb ist (z.B. die ftCashboxIdentification).

Ich würd's hier so stehen lassen wie es ist.